.teamcity/common/extensions.kt

```diff
@@ -100,15 +100,16 @@ fun BuildSteps.checkCleanM2(os: Os = Os.linux) {
     }
 }
 
-fun buildToolGradleParameters(daemon: Boolean = true, isContinue: Boolean = true): List<String> =
+fun buildToolGradleParameters(daemon: Boolean = true, isContinue: Boolean = true, os: Os = Os.linux): List<String> =
     listOf(
         "-PmaxParallelForks=%maxParallelForks%",
         "-s",
         if (daemon) "--daemon" else "--no-daemon",
         if (isContinue) "--continue" else "",
         """-I "%teamcity.build.checkoutDir%/gradle/init-scripts/build-scan.init.gradle.kts"""",
         "-Dorg.gradle.internal.tasks.createops",
-        "-Dorg.gradle.internal.plugins.portal.url.override=%gradle.plugins.portal.url%"
+        // // https://github.com/gradle/gradle-private/issues/2725
+        if (os == Os.macos) "" else "-Dorg.gradle.internal.plugins.portal.url.override=%gradle.plugins.portal.url%"
```

Comment:
I am not convinced this will help, see https://github.com/gradle/gradle-private/issues/2725#issuecomment-533976269.